config/src/utils.rs

```diff
@@ -4,9 +4,24 @@
 use crate::config::NodeConfig;
 use get_if_addrs::get_if_addrs;
 use libra_network_address::{NetworkAddress, Protocol};
-use libra_types::transaction::Transaction;
+use libra_types::{
+    account_address::AccountAddress,
+    transaction::{authenticator::AuthenticationKey, Transaction},
+};
 use std::net::{TcpListener, TcpStream};
 
+pub fn default_validator_owner_auth_key_from_name(name: &[u8]) -> AuthenticationKey {
+    let salt = "validator_owner::";
+    let mut name_in_bytes = salt.as_bytes().to_vec();
+    name_in_bytes.extend_from_slice(name);
+    let hash = libra_crypto::HashValue::sha3_256_of(&name_in_bytes);
+    AuthenticationKey::new(*hash.as_ref())
+}
```

Comment:
I'm assuming this isn't permanent, but just for genesis?  Because otherwise, I could just derive the same key if I know the name right?

Reply:
I'm gonna reach out to AOS, as this wasn't hard, maybe we require all validators and validator operators to have well defined addresses. I had a proposal for this earlier that got bumped due to concerns of complexity...